widget/embedded/src/utils/quote.ts

```diff
@@ -260,6 +265,13 @@ export function generateQuoteWarnings(
   return null;
 }
 
+export function doesQuoteWarningNeedConfirmation(warning: QuoteWarning) {
```

Comment:
For this function, we can choose a name that is clearer and more precise in describing its behavior. For example:

`isQuoteWarningConfirmationRequired`